penguin/integrations/mcp/server_tools/pm.py

```
@@ -0,0 +1,280 @@
+
+def _optional_str_list(value: Any) -> Optional[list[str]]:
+    if value is None:
+        return None
+    if isinstance(value, str):
+        return [value]
+    if isinstance(value, list):
+        return [str(item) for item in value]
+    return None
+
+
+def _parse_status(value: Any) -> Optional[TaskStatus]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)